x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -27,34 +28,52 @@ interface EndpointDetailsTabs {
   id: string;
   name: string;
   content: JSX.Element;
+  route: string;
 }
 
+const EndpointDetailsTab = memo(
+  ({
+    tab,
+    isSelected,
+    handleTabClick,
+  }: {
+    tab: EndpointDetailsTabs;
+    isSelected: boolean;
+    handleTabClick: () => void;
+  }) => {
+    const setUrl = useNavigateByRouterEventHandler(tab.route);
+    const onClick = useCallback(
+      (e: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement, MouseEvent>) => {
+        setUrl(e);
+        handleTabClick();
+      },
+      [setUrl, handleTabClick]
+    );
+    return (
+      <EuiTab onClick={onClick} isSelected={isSelected} key={tab.id} data-test-subj={tab.id}>
```

Comment:
Should we also set the `href` on this? so that user can do right-click open in new tab?

Reply:
Hell yes! Why not! ⭐

Reply:
done 8253ccdd9e4bcb06de841758ad2166f4a2c0b779